templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
+
+{php} if ($GLOBALS['document_drag_drop']) { {/php}
+<form method="post" enctype="multipart/form-data" action="{$WEBROOT}/interface/documents/upload.php?patient_id={php}echo $pid{/php}&parent_id={php}echo $category{/php}" class="dropzone">
```

Comment:
I had to go look at the build and you are right to question. The general_list.php file is handling the dropzone. The upload.php is NOT the view. The upload.php does the work up loading the file and all the rest of the functions for the patient's chart. The upload.php is never seen. It executes silently in the background.